packages/fxa-auth-server/lib/routes/session.js

```diff
@@ -16,7 +15,7 @@ const NodeRendererBindings =
 const SESSION_DOCS = require('../../docs/swagger/session-api').default;
 const DESCRIPTION = require('../../docs/swagger/shared/descriptions').default;
 const HEX_STRING = validators.HEX_STRING;
-const { AccountEventsManager } = require('../account-events');
+import { recordSecurityEvent } from './utils/security-event';
```

Comment:
Seeing this in a js file throws me off even though I think it works.

Reply:
Oh good point, I'll update to be consistent.